packages/nimble-components/src/spinner/specs/README.md

```diff
@@ -0,0 +1,125 @@
+  - If omitted, the default is `medium` (32x32)
+- `appearance-variant`
+    - default (undefined): Standard appearance (light/dark/color themes)
+    - `alternate`: Alternate appearance/color, for the color theme only
+    - Backed by enum `SpinnerAppearanceVariant`
+
+Methods, Events, CSS Classes, CSS Custom Properties: None
+
+### Anatomy 
+
+Contains 2 `div` elements that are animated via CSS animations.
+
+Slot Names, Host Classes, Slotted Content/ Slotted Classes, CSS Parts: None
+
+### Angular integration 
+
+Directive `NimbleSpinnerDirective` targeting selector `nimble-spinner`, with bindings for `size` and `appearance-variant`.
+
+### Blazor integration 
+
+Standard Blazor implementation (`NimbleSpinner` deriving from `ComponentBase`), with bindings for `size` and `appearance-variant`.
+
+### Visual Appearance
+
+[Visual Design spec - Adobe XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/dece308f-79e7-48ec-ab41-011f3376b49b/)
+
+Light / dark themes only have a single appearance variant.  
+The color theme has 2 appearance variants - the "Color UI Alt." one from XD can be accessed with `appearance-variant="alternate"`.
+
+## Implementation
+
+Component implementation will derive from the FAST `FoundationElement`.
+
+**Alternatives Considered:** We could derive from `fast-progress-ring` (radial progress indicator, which supports an indeterminate progress display), however then our component API would also pick up the `min`/`max`/`value` properties for determinate progress (and `paused` to pause the indicator), which we don't plan on supporting at this time. Additionally, the visual appearance of our spinner will be a little different from a radial progress bar, and we still may end up supporting a standard radial progress bar in the future.  
+For reference, the FAST spec for `progress` / `progress-ring` is [here](https://github.com/microsoft/fast/blob/802443ffb2b19a078f9b48f62e6d1a35e3276fb5/packages/web-components/fast-foundation/src/progress/README.md).
+
+### States
+
+Only one component state: visible and animating. Pausing not supported; spinner is expected to be added/removed from the DOM as needed.
+
+### Accessibility
+
+No keyboard states / form integration.
+
+ARIA: Plan to use the [`progressbar role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role) without setting `aria-valuemin` / `aria-valuemax` / `aria-valuenow` (representing indeterminate).
+
+### Globalization
+
+N/A. 
+
+RTL: No special handling / no changes to the spinner animation. Linear indeterminate progress animations are sometimes reversed for RTL, but circular/clockwise ones are usually not (see [Material Design v2, Mirroring Elements](https://m2.material.io/design/usability/bidirectionality.html#mirroring-elements) > *When Not To Mirror*, for one example).
+
+### Security
+
+N/A
+
+### Performance
+
+N/A
+
+### Dependencies
+
+None
+
+### Test Plan
+
+Standard tests planned
```

Comment:
We'll need to figure out how Chromatic tests work for a component that animates. It's possible we'll need an API to pause the animation just so that we can take reproducible snapshots.

Reply:
Looks like if we're using CSS animations [Chromatic should pause those by default](https://www.chromatic.com/docs/animations).